special-interest-groups/sig-ddl/roles-and-organization-management.md

```diff
@@ -28,10 +28,10 @@ Requirements:
 
 Requirements:
 
-* Have at least 40 PRs (at least 10 PRs in tidb repo and at least 8 PRs in parser repo) merged.
-* Have fixed at least 5 issues whose difficulty is medium or above.
+* Have at least 20 PRs (at least 10 PRs in tidb repo and at least 8 PRs in parser repo) merged.
```

Comment:
10 PRs in tidb and 8 PRs in parser? Then it's very limited.

Reply:
Updated.